src/components/ListItemText/ListItemText.js

```diff
@@ -0,0 +1,18 @@
+import { withProps } from 'recompose';
+import styled from 'react-emotion';
+import Text from '../Text';
+
+const ListItemText = styled(
+  withProps({
+    noMargin: true
+  })(Text)
+)`
+  label: list-item-text;
+  padding-left: ${({ theme }) => theme.spacings.mega};
+  flex-grow: 1;
+  overflow: hidden;
```

Comment:
I feel like we should create a style helper for this, if we don't already have one. We keep using this in so many places. Just a note for the future.